### PR TITLE
fix: run more backfill tipsetkey during import chain

### PR DESCRIPTION
**Pull request by Terryhung**

### CODE REVIEW

File: commands/init.go [hunk 1]
```diff
@@ -45,6 +46,13 @@ var InitCmd = &cli.Command{
 			EnvVars:     []string{"LILY_SNAPSHOT"},
 			Destination: &initFlags.importSnapshot,
 		},
+		&cli.IntFlag{
+			Name:        "backfill-tipsetkey-range",
+			Usage:       "Determine the extent of backfilling from the head.",
+			EnvVars:     []string{"LILY_BACKFILL_TIPSETKEY_RANGE"},
+			Value:       1800,
```
Comment (birdychang) [inline commands/init.go:53]: our snapshot range is 2890 epochs. Should we make the default something like 3000 or 3600 to ensure we always backfill more than a day?

File: commands/util/import.go [hunk 2]
```diff
@@ -189,6 +190,11 @@ func ImportChain(ctx context.Context, r repo.Repo, fname string, snapshot bool)
 		return fmt.Errorf("importing chain failed: %w", err)
 	}
 
+	err = backfillMoreEpochsForTipsetKey(ctx, ts, cst, backfillTipsetkeyRange)
```
Comment (birdychang) [inline commands/util/import.go:193]: Add comment to explain `cst.Import()` only backfills 1800 epochs.
Make the name shorter `backfillTipsetKey` .

**Review by birdychang — APPROVED** [review]

File: commands/init.go (hunk 1 above)
Comment (Terryhung) [reply commands/init.go]: changed

File: commands/util/import.go (hunk 2 above)
Comment (Terryhung) [reply commands/util/import.go]: added more comment!

File: commands/util/import.go [hunk 3]
```diff
@@ -189,6 +190,14 @@ func ImportChain(ctx context.Context, r repo.Repo, fname string, snapshot bool)
 		return fmt.Errorf("importing chain failed: %w", err)
 	}
 
+	// The cst.Import function will only backfill 1800 epochs of tipsetkey,
+	// as determined by the value specified in TipsetkeyBackfillRange.
```
Comment (birdychang) [inline commands/util/import.go:194]: this line might be confusing since we have `backfillTipsetKeyRange`. No need to mention the variable in `cst.import`.

**Review by birdychang — APPROVED** [review]

File: commands/util/import.go (hunk 3 above)
Comment (Terryhung) [reply commands/util/import.go]: removed

**Pull request merged**